### PR TITLE
OT-Redis

### PR DESCRIPTION
**Pull request by nmohantyctr**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/databases/open-telemetry/redis.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+description: Documentation for Open Telemetry App for Redis.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/databases/redis.png')} alt="Thumbnail icon" width="50"/>
+
+[Redis](https://redis.io/docs/about/) is an in-memory data structure implementing a distributed, in-memory key-value database with optional durability.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:13]: I'd rephrase this as:
Redis is a distributed key-value database that operates in memory and has the option for durability. It implements a data structure that is stored in the memory of the computer and can be accessed quickly, making it useful for applications that require high performance.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+The Sumo Logic App for Redis helps you monitor the Redis database cluster. The preconfigured dashboards provide detailed analysis based on logs. The panels provide details such as RDBMemory Usage, events, RDB and AOF events.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:15]:
```suggestion
The Sumo Logic App for Redis helps you monitor the Redis database cluster. The preconfigured dashboards provide detailed analysis based on logs. The panels provide details such as RDBMemory Usage, events, RDB, and AOF events.
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+Redis logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/schematics-redis.png')} alt="Redis Schematics" />
+
+##Redis Log Types
```
Comment (sumoanema) [inline docs/integrations/databases/open-telemetry/redis.md:21]: The heading is not appearing properly when I render it. I thing the problem is since there is no space between ## and the actual text.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+This Sumo Logic App for Redis supports Logs for Redis in Cluster or Standalone mode. The Redis logs are generated in files as configured in the cluster node configuration files in `/etc/redis/redis.conf`. For more details on Redis logs, [click here](https://redislabs.com/ebook/part-2-core-concepts/chapter-5-using-redis-for-application-support/5-1-logging-to-redis/).
+
+##Fields creation in Sumo Logic for Redis
```
Comment (sumoanema) [inline docs/integrations/databases/open-telemetry/redis.md:25]: Same problem. Heading not appearing correctly.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+Click on the "Download YAML File" button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/configuration-screenshot.png')} alt="Configuration" />
+
+###Step3: Sending logs to Sumo
+
+Once you have the yaml file downloaded in step 2, please follow the below steps based on your environment
+
+####Linux
```
Comment (sumoanema) [inline docs/integrations/databases/open-telemetry/redis.md:90]: Same problem. Heading not appearing correctly.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
+id: redis
+title: Sumo Logic Open Telemetry App for Redis
+sidebar_label: Redis
+description: Documentation for Open Telemetry App for Redis.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:4]: Can you please write the description that should say about the content of this doc?

File: docs/integrations/databases/open-telemetry/redis.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+###Step3: Sending logs to Sumo
+
+Once you have the yaml file downloaded in step 2, please follow the below steps based on your environment
+
+####Linux
+
+1.  Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Redis instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the below command `sudo systemctl restart otelcol-sumo`
+
+####Mac
```
Comment (sumoanema) [inline docs/integrations/databases/open-telemetry/redis.md:95]: Same problem. Heading not appearing correctly.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+This section provides instructions for configuring log collection for Redis running on a non-Kubernetes environment for the Sumo Logic App for Redis. By default, Redis logs are stored in a log file.
+
+Follow the instructions to set up log collection:
+
+1. To configure the Redis log file, locate your local `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` configuration file in the database directory. By default, Redis logs are stored in `/var/log/redis/redis-server.log`.
+
+After determining the location of conf file modify the `redis.conf` configuration file logging parameters:
+
+  *   Open `redis.conf` configuration file in a text editor.
```
Comment (sumoanema) [inline docs/integrations/databases/open-telemetry/redis.md:44]: All the points are rendered to be in the same hierarchy. Could you please refer the gdoc for the point hierarchy

File: docs/integrations/databases/open-telemetry/redis.md (hunk 1 above)
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:13]: I'd rephrase this as:
[Redis](https://redis.io/docs/about/) is an in-memory data structure that implements a distributed, in-memory key-value database with durability options.

File: docs/integrations/databases/open-telemetry/redis.md (hunk 6 above)
Comment (JainM6) [reply docs/integrations/databases/open-telemetry/redis.md:4]: OpenTelemetry should be one word.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+2.  Restart the otelcol-sumo process using the below command `sudo systemctl restart otelcol-sumo`
+
+####Mac
+
+1.  Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Redis instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the command `otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --conf "glob:/etc/otelcol-sumo/conf.d/*.yaml"`
+
+After the successful execution of the above command, Sumo will start receiving the data from your host machine.
+
+This will install the app to your Sumo Logic Org. The app consists of Dashboards.
```
Comment (sumoanema) [inline docs/integrations/databases/open-telemetry/redis.md:102]: Please add "Press Next" at the starting of this line.

**Review by sumoanema — DISMISSED** [review]

File: docs/integrations/databases/open-telemetry/redis.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+
+The Sumo Logic App for Redis helps you monitor the Redis database cluster. The preconfigured dashboards provide detailed analysis based on logs. The panels provide details such as RDBMemory Usage, events, RDB and AOF events.
+
+Redis logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/schematics-redis.png')} alt="Redis Schematics" />
+
+##Redis Log Types
+
+This Sumo Logic App for Redis supports Logs for Redis in Cluster or Standalone mode. The Redis logs are generated in files as configured in the cluster node configuration files in `/etc/redis/redis.conf`. For more details on Redis logs, [click here](https://redislabs.com/ebook/part-2-core-concepts/chapter-5-using-redis-for-application-support/5-1-logging-to-redis/).
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:23]:
```suggestion
This Sumo Logic App for Redis supports logs for Redis in Cluster or Standalone mode. The Redis logs are generated in files as configured in the cluster node configuration files in `/etc/redis/redis.conf`. For more details on Redis logs, [click here](https://redislabs.com/ebook/part-2-core-concepts/chapter-5-using-redis-for-application-support/5-1-logging-to-redis/).
```

File: docs/integrations/databases/open-telemetry/redis.md (hunk 4 above)
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:25]: I'd say :
Creating Fields in Sumo Logic for Redis

File: docs/integrations/databases/open-telemetry/redis.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+
+##Fields creation in Sumo Logic for Redis
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/), which will be created as part of Redis App install if not already present.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:27]:
```suggestion
Following are the [Fields](https://help.sumologic.com/docs/manage/fields/), which will be created as part of the Redis App install if not already present.
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+
+**db.cluster.name** - User configured. Enter a name to identify this Redis cluster. This cluster name will be shown in the Sumo Logic dashboards.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:29]: I'd suggest we should create a table and use the single quotes for fields or parameters.
Like this. `db.cluster.name` 

Bold should be used when we talk about the UI elements.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+**db.system** - Has fixed value of redis
+**deployment.environment** - User configured. This is the deployment environment where the Redis cluster resides. For example: dev, prod or qa.
+**sumo.datasource** - has fixed value of redis
+
+##Prerequisite
+
+This section provides instructions for configuring log collection for Redis running on a non-Kubernetes environment for the Sumo Logic App for Redis. By default, Redis logs are stored in a log file.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:36]:
```suggestion
This section provides instructions for configuring log collection for Redis running on a non-Kubernetes environment for the Sumo Logic App for Redis. By default, Redis logs are stored in a log file.
```
```suggestion
This section provides instructions for configuring log collection for Redis running on a non-Kubernetes environment. By default, Redis logs are stored in a log file.
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+
+Follow the instructions to set up log collection:
+
+1. To configure the Redis log file, locate your local `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` configuration file in the database directory. By default, Redis logs are stored in `/var/log/redis/redis-server.log`.
+
+After determining the location of conf file modify the `redis.conf` configuration file logging parameters:
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:42]:
```suggestion
After determining the location of the `conf file`, modify the `redis.conf` configuration file logging parameters:
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 15]
```diff
@@ -0,0 +1,135 @@
+id: redis
```
Comment (kimsauce) [inline docs/integrations/databases/open-telemetry/redis.md:1]: Missing the three dashes above the frontmatter

```suggestion
---
id: redis
```

File: docs/integrations/databases/open-telemetry/redis.md (hunk 14 above)
Comment (JainM6) [reply docs/integrations/databases/open-telemetry/redis.md:42]: This should point no. 2.

File: docs/integrations/databases/open-telemetry/redis.md (hunk 3 above)
Comment (kimsauce) [inline docs/integrations/databases/open-telemetry/redis.md:21]:
```suggestion
## Redis Log Types
```

File: docs/integrations/databases/open-telemetry/redis.md (hunk 8 above)
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:44]: Point 3.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+This section provides instructions for configuring log collection for Redis running on a non-Kubernetes environment for the Sumo Logic App for Redis. By default, Redis logs are stored in a log file.
+
+Follow the instructions to set up log collection:
+
+1. To configure the Redis log file, locate your local `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` configuration file in the database directory. By default, Redis logs are stored in `/var/log/redis/redis-server.log`.
+
+After determining the location of conf file modify the `redis.conf` configuration file logging parameters:
+
+  *   Open `redis.conf` configuration file in a text editor.
+  *   Set the following config parameters:
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:45]: Can you please correct the numbering here? And these should have numbered bullets.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 17]
```diff
@@ -0,0 +1,135 @@
...
+
+After the successful execution of the above command, Sumo will start receiving the data from your host machine.
+
+This will install the app to your Sumo Logic Org. The app consists of Dashboards.
+
+Results will be available after a while. It is important to note that each panel slowly and automatically fills with data matching the time range query received since the panel was created. Your data should appear in the dashboard within 20 minutes, and you will see complete graphs and maps.
+
+### Sample Log Messages in Non-Kubernetes environments
+
+5275:S 17 Mar 2021  19:13:38.138 * MASTER <-> REPLICA sync: Finished with success
```
Comment (kimsauce) [inline docs/integrations/databases/open-telemetry/redis.md:108]: please format as code

File: docs/integrations/databases/open-telemetry/redis.md [hunk 18]
```diff
@@ -0,0 +1,135 @@
...
+Query String
+db.cluster.name=* sumo.datasource="redis"
+| json auto maxdepth 1 nodrop
+| if  (isEmpty(log), _raw, log)  as message
+| count by message
+| limit  100
+| fields message
+```
+
+##Viewing Redis Dashboards
```
Comment (kimsauce) [inline docs/integrations/databases/open-telemetry/redis.md:124]: missing space after ##

File: docs/integrations/databases/open-telemetry/redis.md [hunk 19]
```diff
@@ -0,0 +1,135 @@
...
+Follow the instructions to set up log collection:
+
+1. To configure the Redis log file, locate your local `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` configuration file in the database directory. By default, Redis logs are stored in `/var/log/redis/redis-server.log`.
+
+After determining the location of conf file modify the `redis.conf` configuration file logging parameters:
+
+  *   Open `redis.conf` configuration file in a text editor.
+  *   Set the following config parameters:
+    *   Specify the server verbosity level. The value loglevel in conf file can be set to one of the following:
+      *   debug (a lot of information, useful for development/testing)
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:47]: I think it's better if we remove this parenthesis, and define the terms, a one-line description is enough.

Comment (JainM6) [reply docs/integrations/databases/open-telemetry/redis.md:47]: The same should be followed in 47, 48,49, 50

File: docs/integrations/databases/open-telemetry/redis.md [hunk 20]
```diff
@@ -0,0 +1,135 @@
...
+      *   verbose (many rarely useful info, but not a mess like the debug level)
+      *   notice (moderately verbose, what you want in production probably)
+      *   warning (only very important/critical messages are logged) loglevel notice
+    *   Specify the log file name. Also the empty string can be used to force Redis to log on the standard output. Note that if you use the standard output for logging but daemonize, logs will be sent to `/dev/null` logfile
+  *   Save the `redis.conf` file and restart the redis server using command : `sudo  service redis-server restart`
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:52]:
```suggestion
  *   Save the `redis.conf` file and restart the Redis server using the command: `sudo  service redis-server restart`
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 21]
```diff
@@ -0,0 +1,135 @@
...
+
+Once the logs are configured to be written to a local file, follow the below step to configure collection in Sumo.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:54]:
```suggestion
Once the logs are configured to be written to a local file, follow the steps below to configure the collection in Sumo Logic.
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 22]
```diff
@@ -0,0 +1,135 @@
...
+
+##Collection Configuration & App installation
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:56]:
```suggestion
## Collection Configuration & App installation
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Formatting + Alekh and Megha's corrections

File: docs/integrations/databases/open-telemetry/redis.md (hunk 22 above)
Comment (JainM6) [reply docs/integrations/databases/open-telemetry/redis.md:56]: I'd rephrase the heading as:

## Configuring Collection 
I don't see App Installation in this doc. If we don't have the app installation steps, we should remove this from the heading.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 23]
```diff
@@ -0,0 +1,135 @@
...
+
+As part of setting up the collection process and app installation user can select the App from App Catalog and click Install App. Please follow the steps below:
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:58]:
```suggestion
As part of setting up the collection process and app installation, you can select the App from **App Catalog** and click **Install App**.  Follow the steps below:
```

Comment (JainM6) [reply docs/integrations/databases/open-telemetry/redis.md:58]: We avoid using "please" in the customer-facing user documentation.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 24]
```diff
@@ -0,0 +1,135 @@
...
+
+### Step1: Set up Collector
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:60]:
```suggestion
### Step 1: Set up Collector
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 25]
```diff
@@ -0,0 +1,135 @@
...
+
+If you want to use an existing OpenTelemetry Collector, then this step can be skipped by selecting the option of using an existing Collector.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:62]: This should come in note.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 26]
```diff
@@ -0,0 +1,135 @@
...
+
+If you want to create a new Collector, please select the "Add a new Collector" option.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:64]:
```suggestion
If you want to create a new Collector, select the **Add a new Collector** option.
```

Comment (JainM6) [reply docs/integrations/databases/open-telemetry/redis.md:64]: Numbering is missing..

File: docs/integrations/databases/open-telemetry/redis.md [hunk 27]
```diff
@@ -0,0 +1,135 @@
...
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command that can be executed in the machine that needs monitoring. Once executed, it will install the Sumo Logic OpenTelemetry Collector agent.
+
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:69]:
```suggestion
```

Comment (JainM6) [reply docs/integrations/databases/open-telemetry/redis.md:69]: Can you please remove the in-line extra spaces?

File: docs/integrations/databases/open-telemetry/redis.md [hunk 28]
```diff
@@ -0,0 +1,135 @@
...
+### Step1: Set up Collector
+
+If you want to use an existing OpenTelemetry Collector, then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector, please select the "Add a new Collector" option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command that can be executed in the machine that needs monitoring. Once executed, it will install the Sumo Logic OpenTelemetry Collector agent.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:68]:
```suggestion
This will generate a command that can be executed in the machine that needs monitoring. Once executed, it will install the Sumo Logic OpenTelemetry Collector agent. <br/> <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/collector-screenshot.png')} alt="Collector" />
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 29]
```diff
@@ -0,0 +1,135 @@
...
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/collector-screenshot.png')} alt="Collector" />
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:70]: You need to remove this, coz we have added the image along with the previous step.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 30]
```diff
@@ -0,0 +1,135 @@
...
+
+###Step2: Configure integration
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:72]:
```suggestion
### Step2: Configuring Integration
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 31]
```diff
@@ -0,0 +1,135 @@
...
+
+Open Telemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file with all the details concerning the data that needs to be collected. For example, it specifies the location of a log file that is read and sent to Sumo Logic platform.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:74]:
```suggestion
OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file with all the details concerning the data that needs to be collected. For example, it specifies the location of a log file that is read and sent to the Sumo Logic platform.
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 32]
```diff
@@ -0,0 +1,135 @@
...
+
+In this step we will be configuring the yaml required for Redis Collection.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:76]:
```suggestion
In this step, we will be configuring the yaml required for Redis Collection.
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 33]
```diff
@@ -0,0 +1,135 @@
...
+
+The log file path configured to capture redis logs must be given here.
+
+The files are typically located in `/var/log/redis/redis-server.log`. If you are using a customized path, check the `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` file for this information.
+
+Click on the "Download YAML File" button to get the yaml file.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:82]:
```suggestion
Click on the **Download YAML File** button to get the yaml file.
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 34]
```diff
@@ -0,0 +1,135 @@
...
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/configuration-screenshot.png')} alt="Configuration" />
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:84]: remove this

File: docs/integrations/databases/open-telemetry/redis.md [hunk 35]
```diff
@@ -0,0 +1,135 @@
...
+
+###Step3: Sending logs to Sumo
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:86]:
```suggestion
### Step 3: Sending logs to Sumo Logic
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 36]
```diff
@@ -0,0 +1,135 @@
...
+
+Once you have the yaml file downloaded in step 2, please follow the below steps based on your environment
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:88]:
```suggestion
Once you have the yaml file downloaded in step 2, please follow the below steps based on your environment.
```

Comment (JainM6) [reply docs/integrations/databases/open-telemetry/redis.md:88]: You should link Step 2 here.

File: docs/integrations/databases/open-telemetry/redis.md (hunk 5 above)
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:90]:
```suggestion
#### Linux
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 37]
```diff
@@ -0,0 +1,135 @@
...
+
+####Linux
+
+1.  Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Redis instance which needs to be monitored.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:92]:
```suggestion
1.  Copy the yaml at **/etc/otelcol-sumo/conf.d/** folder in the Redis instance which needs to be monitored.
```

File: docs/integrations/databases/open-telemetry/redis.md (hunk 7 above)
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:95]:
```suggestion
#### Mac
```

File: docs/integrations/databases/open-telemetry/redis.md [hunk 38]
```diff
@@ -0,0 +1,135 @@
...
+2.  Restart the otelcol-sumo process using the below command `sudo systemctl restart otelcol-sumo`
+
+####Mac
+
+1.  Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Redis instance which needs to be monitored.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:97]: This is the folder name, it should be in bold.

File: docs/integrations/databases/open-telemetry/redis.md [hunk 39]
```diff
@@ -0,0 +1,135 @@
...
+2.  Restart the otelcol-sumo process using the command `otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --conf "glob:/etc/otelcol-sumo/conf.d/*.yaml"`
+
+After the successful execution of the above command, Sumo will start receiving the data from your host machine.
```
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:100]: I'd rephrase this as:
Sumo Logic will start receiving data from your host machine when the above command is successfully executed.

File: docs/integrations/databases/open-telemetry/redis.md (hunk 18 above)
Comment (JainM6) [inline docs/integrations/databases/open-telemetry/redis.md:124]:
```suggestion
## Viewing Redis Dashboards
```

**Review by JainM6 — DISMISSED** [review]
Formatting changes.

File: sidebars.ts [hunk 40]
```diff
@@ -1848,6 +1848,15 @@ integrations: [
...
         items: [
+          {
+            type: 'category',
+            label: 'OpenTelemetry',
+            collapsible: true,
+            collapsed: true,
+            items: [
+              'integrations/databases/opentelemetry/redis',
+            ],
+          },
```
Comment (kimsauce) [inline sidebars.ts:1859]:
```suggestion
```

**Review by kimsauce — APPROVED** [review]
